### DISCUSSION TITLE
Any benefit to have statically imported images under public folder?

### DISCUSSION THREAD

**Discussion by samuliasmala** [vercel/next.js#40578] [Help] [answered]
If I import images statically is there any benefit placing them under `/public` folder vs another folder (pls. see the example below)? I know that if you're using a path string then you must place them under `/public` but does it make any difference if using static imports?

```ts
import Image from 'next/image';

import image1 from '../assets/image.png';
import image2 from '../public/images/image.png';

<Image src={image1} width={150} height={150} />
<Image src={image2} width={150} height={150} />
```

**Comment by icyJoseph** [comment, marked as answer]
Well, the image from the public folder can be accessed at `https://<your-domain>/images/image.png`. 

> Note that both cases can be accessed, IIRC though the `_next/image?url=<internal-encoded-path>`, but if you decode the path I think you cannot really reach them. 

Other than that I think it is almost the same, then there's also the shorthand:

```jsx
 <Image src="/images/image.png" width={150} height={150} />
```

Which unlike your other two examples, resolves to a simpler `src` on the `<img>` tag.

**Reply by samuliasmala** [reply]
Thanks for the answer! Some of my deployments from the same code base use a basepath so I would need to prefix the path string with a environment variable where the basepath is stored. Thus I prefer the static imports where no prefixing is required,
